NEWS

```diff
@@ -31,6 +31,15 @@ CHANGES WITH 257 in spe:
           by default when combined with --scope, will be changed in a future
           release to be enabled by default.
 
+        * systemd-logind now always obeys inibitor locks, where previously it
```

Comment:
typo, btw

Reply:
mmh where? I am probably blind but cannot see it

Reply:
never mind, spotted on the 5th re-reading